_ont/ont-odi-realtek-dfp-34x-2c2.md

```diff
@@ -80,6 +80,7 @@ The stick has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be a
 {% include alert.html content="Some USB TTL adapters label TX and RX pins the other way around: try to swap them if the connection doesn't work." alert="Note"  icon="svg-warning" color="yellow" %}
 
 {% include_relative ont-luna-sdk-useful-commands.md flash='flash' ploam='hex' customSwVersionAlert='This needs the `OMCI_OLT_MODE` value to be set to 3 and firmware 220530 or 220923 modded by @stich86' speedLan='1234567' %}
```

Comment:
```suggestion
{% include_relative ont-luna-sdk-useful-commands.md flash='flash' ploam='hex' customSwVersionAlert='This needs the `OMCI_OLT_MODE` value to be set to 3 and firmware 220530 or 220923 modded by @stich86 or if you don't want to replace firmware to change software version, set `OMCI_OLT_MODE` value to `21`. This will force to use your own settings from the XML file' speedLan='1234567' %}
```

Reply:
and remove the  alert.html